__tests__/api.test.js

```diff
@@ -53,33 +53,91 @@ describe("GET:/api/articles", () => {
       .then(({ body }) => {
-        expect(body.msg).toBe('bad request')
+        expect(body.msg).toBe("bad request");
       });
   });
   test("400: invalid id (string)", () => {
     return request(app)
       .get("/api/articles/sdsc")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('bad request')
+        expect(body.msg).toBe("bad request");
+      });
+  });
+});
+
+describe("PATCH:/api/articles:article_id/", () => {
+  test("200:update votes on article by id (returns updated object)", () => {
+    return request(app)
+      .patch("/api/articles/6")
+      .expect(200)
+      .send({ inc_votes: 100 })
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            author: "icellusedkars",
+            title: "A",
+            topic: "mitch",
+            article_id: 6,
+            body: "Delicious tin of cat food",
+            created_at: "2020-10-18T01:00:00.000Z",
+            votes: 100,
+          })
+        );
+      });
+  });
+  test("404: invalid id doesnt exist(still a number)", () => {
+    return request(app)
+      .patch("/api/articles/34890384")
+      .expect(404)
+      .send({ inc_votes: 100 })
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+  test("400: invalid id (string)", () => {
+    return request(app)
+      .patch("/api/articles/sdsc")
+      .send({ inc_votes: 100 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("400: body has object but no inc_votes key", () => {
+    return request(app)
+      .patch("/api/articles/sdsc")
+      .expect(400)
+      .send({banana : 100})
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("400: body has inc_votes, but with other keys", () => {
+    return request(app)
+      .patch("/api/articles/sdsc")
+      .expect(400)
+      .send({inc_votes : 100,banana:'100'})
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
       });
   });
 });
 
 describe("GET /api/users", () => {
-    test("200: responds with an array of objects with slug and description props ", () => {
-      return request(app)
-        .get("/api/users")
-        .expect(200)
-        .then(({ body }) => {
-          expect(Array.isArray(body.users)).toBe(true);
-          expect(body.users.length > 0).toBe(true);
-  
-          body.users.forEach((obj) => {
-            expect(obj).toHaveProperty("username");
-            expect(obj).toHaveProperty("name");
-            expect(obj).toHaveProperty("avatar_url");
-          });
+  test("200: responds with an array of objects with slug and description props ", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
+        expect(body.users.length > 0).toBe(true);
```

Comment:
Don't know if someone has already approved this, but its on this PR request. 
You need to check the array has the correct number of users in it, not just greater than 0. 
At the moment this would pass even if you only got back a single user in the array.